test/e2e/sagemaker/__init__.py

```diff
@@ -4,22 +4,45 @@
 # not use this file except in compliance with the License. A copy of the
 # License is located at
 #
-#	 http://aws.amazon.com/apache2.0/
+# 	 http://aws.amazon.com/apache2.0/
```

Comment:
???

Reply:
black tool, looks like the template had spacing issue